gqueries/modules/factsheet/energetic_final_demand/factsheet_energetic_final_demand_gas_households_buildings.gql

```diff
@@ -0,0 +1,7 @@
+- query =
+    SUM(
+      V(INTERSECTION(USE(energetic),SECTOR(households)),"input_of_network_gas + input_of_hydrogen").sum,
+      V(INTERSECTION(USE(energetic),SECTOR(buildings)),"input_of_network_gas + input_of_hydrogen").sum
```

Comment:
I think this query is incorrect. `demand_gas_other` uses  

```ruby
INTERSECTION(USE(energetic), G(final_demand_group))
```

(energetic nodes in the final demand group), while this query uses:

```ruby
INTERSECTION(USE(energetic), SECTOR(households))
```

(**all** energetic nodes in households/buildings).

This means that "final_demand_gas_households_buildings" can return a higher value than the first part of the query in "final_demand_gas_other", which in turn results in that query returning a negative (beta scenario 808292):

<img width="322" alt="screen shot 2018-02-20 at 10 55 20" src="https://user-images.githubusercontent.com/4383/36420332-94134334-162c-11e8-9e2b-4f74bb35b38f.png">

```ruby
INTERSECTION(USE(energetic),SECTOR(households)).length
# => 122

INTERSECTION(INTERSECTION(USE(energetic),SECTOR(households)), G(final_demand_group)).length
# => 7
```

I suspect that what you actually want is something like:

```ruby
INTERSECTION(INTERSECTION(USE(energetic), SECTOR(households)), G(final_demand_group))
```

It might best to define a new query which returns the energetic final demand nodes:

```ruby
INTERSECTION(G(final_demand_group), USE(energetic))
```

... and then reuse that in the two gas queries:

```ruby
SUM(
  V(INTERSECTION(Q(factsheet_energetic_gas_converters),SECTOR(households)),"input_of_network_gas + input_of_hydrogen"),
  V(INTERSECTION(Q(factsheet_energetic_gas_converters),SECTOR(buildings)),"input_of_network_gas + input_of_hydrogen")
)

V(Q(factsheet_energetic_gas_converters),"input_of_network_gas + input_of_hydrogen + input_of_lng + input_of_compressed_network_gas").sum - Q(factsheet_energetic_final_demand_gas_households_buildings)
```

Reply:
Great catch @antw! I am certain that was the idea 👍

Reply:
Fixed and pushed to the new `factsheet-queries-chart-2` branch